### PR TITLE
Development login frontend fix

### PR DESCRIPTION
**Pull request by aliasaddik**
Login handling in front-end
adding authorization to axios
adding private routes for every type of user

### CODE REVIEW

File: client/src/Routes.svelte [hunk 1]
```diff
@@ -1,62 +1,65 @@
...
+  import LoggedRoute from './routes/LoggedRoute.svelte';
+  // import AdminRoute from "./routes/AdminRoute.svelte";
+  // import Supervisor from './routes/Supervisor.svelte'
+  import Login from './pages/Login.svelte';
+  // import { setTheme } from './services/utils/theme';
+  // import isAuthenticated from './services/authentication/IsAuthenticated';
+  // import { onMount } from 'svelte';
   import Error from './pages/Error.svelte';
-  import { UserStore } from './stores';
+  // import { UserStore } from './stores';
```
Comment (Mahmoud-Emad) [inline client/src/Routes.svelte:18]: Please remove unused comments.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -20,10 +20,9 @@ jobs:
           python-version: 3.8
 
       - name: Install Python dependencies
-        run: pip install black flake8
+        run: pip install flake8
```
Comment (Mahmoud-Emad) [inline .github/workflows/lint.yml:23]: why we use pip while we can use `poetry add`

File: client/src/Routes.svelte [hunk 3]
```diff
@@ -1,62 +1,65 @@
...
+  //       status: 302,
+  //       redirect: '/login',
+  //     };
+  //   } else {
+  //     return {
+  //       status: 200,
+  //       redirect: '/calendar',
+  //     };
+  //   }
+  // });
```
Comment (Mahmoud-Emad) [inline client/src/Routes.svelte:45]: too much unused comments :D

**Comment by omar-sherif9992** [comment]
- fix unhandled scenarios
- changed the library to svelte-navigatior 
- svelte-routing : is so limited capability and had bugs  cause of it in another areas

**Pull request merged**